src/screens/People/People.js

```diff
@@ -399,7 +373,7 @@ class PeopleScreen extends React.Component<Props, State> {
     }, 1000);
   };
 
-  handleReferralBannerPress = () => {
+  handleInvitePress = () => {
     const { navigation, user } = this.props;
```

Comment:
one nitpick: we have this code already in 3 different places and there is a high possibility we'll need it in another places, is there any possibility to move it to utils? or at least to the InviteBanner component?

Reply:
Hm, this could be made into an action. I'll take a look :)

Reply:
Done :)